doc-QuickStart_OpenStack_Provider_Guide/topics/Provisioning_Requests.adoc

```diff
@@ -0,0 +1,45 @@
+[[provisioning-Instances]]
+=== Provisioning an OpenStack Instance from an Image
+
+Cloud instances follow the same process (Request, Approval, Deployment) as a standard virtual machine from virtualization infrastructure. First, a user makes a request for instances and specifies the image, tags, availability zone and hardware profile flavor. Second, the request goes through the approval phase. Finally, _{product-title}_ executes the request.
+		
+This procedure describes how to provision an OpenStack instance from an image:
+
+. Navigate to 'Menu > Clouds > Instances'.
+. Click image:2007.png[](*Lifecycle*), then click image:1848.png[](*Provision Instances*).
+. Select an OpenStack image from the list presented. These images must be available on your OpenStack provider.
+. Click *Continue*.
+. On the *Request* tab, enter information about this provisioning request. In *Request Information*, type in at least a first and last name and an email address. This email is used to send the requester status emails during the provisioning process for items such as auto-approval, quota, provision complete, retirement, request pending approval, and request denied. The other information is optional. If the _{product-title}_ Server is configured to use LDAP, you can use the *Look Up* button to populate the other fields based on the email address.
++
+[NOTE]
+======
+Parameters with a * next to the label are required to submit the provisioning request.
+======
++
+. Click the *Purpose* tab to select the appropriate tags for the provisioned instance.
+. Click the *Catalog* tab for basic instance options.
+.. To change the image to use as a basis for the instance, select it from the list of images.
+.. Select the *Number of Instances* to provision.
+.. Type an *Instance Name* and *Instance Description*.
+. Click the *Environment* tab to select the instance's *Tenant*, *Availabilty Zones*, *Cloud Network*, *Security Groups*, and *Public IP Address*. If no specific Tenant is required, select the *Choose Automatically* checkbox.
```

Comment:
Edit:
I think 'tenant" should be lowercase in "If no specific Tenant is required,"